src/storage/kv/mock_engine.rs

```diff
@@ -13,7 +13,15 @@ use std::sync::{Arc, Mutex};
 #[derive(Clone)]
 pub struct MockEngine {
     base: RocksEngine,
-    expected_modifies: Arc<ExpectedWriteList>,
+    expected_modifies: Option<Arc<ExpectedWriteList>>,
```

Comment:
Why make it an `Option`?

Reply:
To indicate whether we need to check it.